test/integration/apiserver/oidc/oidc_test.go

```diff
@@ -432,6 +432,8 @@ jwt:
     url: %s
     audiences:
     - %s
+    - another-audience
```

Comment:
This can be a follow up but I would like to see an integration test that relies on a different audience to pass.